packages/ag-charts-community/src/chart/series/seriesTypes.ts

```diff
@@ -54,10 +54,11 @@ export interface ISeries<TDatum, TProps, TLabel = TDatum> {
  * Processed series datum used in node selections,
  * contains information used to render pie sectors, bars, markers, etc.
  */
-export interface SeriesNodeDatum {
+export interface SeriesNodeDatum<I> {
```

Comment:
nit: Consider using `SeriesNodeDatum<I = unknown>` to avoid needed to write `SeriesNodeDatum<unknown>` everywhere.

Reply:
If it's ok, I'll revisit this after everything is migrated over. I suspect this will be a required property, but if I'm wrong, I can always update the code later

Reply:
Okay. SGTM